rust/crd/src/lib.rs

```diff
@@ -263,14 +269,45 @@ impl FromStr for TrinoRole {
 #[serde(rename_all = "camelCase")]
 pub struct TrinoClusterStatus {}
 
-#[derive(Clone, Debug, Default, Deserialize, Eq, JsonSchema, PartialEq, Serialize)]
+#[derive(Clone, Debug, Default, Deserialize, Merge, JsonSchema, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct TrinoStorageConfig {
+    #[serde(default)]
+    pub data: PvcConfig,
+}
+
+#[derive(Clone, Debug, Default, Deserialize, JsonSchema, PartialEq, Serialize)]
 #[serde(rename_all = "camelCase")]
 pub struct TrinoConfig {
     // config.properties
     pub query_max_memory: Option<String>,
     pub query_max_memory_per_node: Option<String>,
     // log.properties
     pub log_level: Option<String>,
+    // misc
+    pub resources: Option<Resources<TrinoStorageConfig, NoRuntimeLimits>>,
+}
+
+impl TrinoConfig {
+    fn default_resources() -> Resources<TrinoStorageConfig, NoRuntimeLimits> {
+        Resources {
+            cpu: CpuLimits {
+                min: Some(Quantity("100m".to_owned())),
+                max: Some(Quantity("1".to_owned())),
+            },
+            memory: MemoryLimits {
+                limit: Some(Quantity("2Gi".to_owned())),
+                runtime_limits: NoRuntimeLimits {},
+            },
+            storage: TrinoStorageConfig {
+                data: PvcConfig {
+                    capacity: Some(Quantity("2Gi".to_owned())),
```

Comment:
The docs say 1Gi. Somehow the test see to default to 1Gi as well...

Reply:
I think the crd was not updated correctly or something and therefore ignored...will invstigate